ui/src/styles/theme.ts

```diff
@@ -5,7 +5,7 @@ import { StyleVariables } from "src/domain";
 import variables from "src/styles/variables.module.scss";
 import { StrictSchema } from "src/utils/types";
 
-export const parseStyleVariables = StrictSchema<StyleVariables>()(
+const parseStyleVariables = StrictSchema<StyleVariables>()(
```

Comment:
If we only ever use this once, should it just be in `parsedStyleVariables`? Ditto for other instances like this.

Reply:
I'm not sure I understand the comment. If you refer to moving the type `StyleVariables` into this module I can't agree more. Done. If you meant something else, can you please reformulate the comment?

Reply:
Ahh! oc we can parse the value directly after defining the parser.

Also, we don't need to use `safeParse` to throw a string in the `if` block below. It's better to use `parse` and let `zod` throw with a more meaningful error. A different discussion is whether we should throw like this, but I was not involved in this implementation and can't tell for sure why all this was architectured this way.

Changed.